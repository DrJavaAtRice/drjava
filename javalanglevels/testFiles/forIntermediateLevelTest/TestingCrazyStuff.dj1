//This class tests autoboxing and unboxing.
class TestingCrazyStuff {
 static Object o = 5; 
 static Object b = true;
 
 public Object myMethod() {
   if (1<2) {
     return 5;
   }
   else {
     return true;
   }
 }
}