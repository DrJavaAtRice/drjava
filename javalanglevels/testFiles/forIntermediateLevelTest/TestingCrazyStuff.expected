// Language Level Converter line number map: dj*->java. Entries: 14
//     1->4         2->5         3->6         4->7         5->8         6->9         7->10        8->11   
//     9->12       10->13       11->14       12->15       13->16       14->17   
//This class tests autoboxing and unboxing.
class TestingCrazyStuff {
 public final static Object o = 5; 
 public final static Object b = true;
 
 public Object myMethod() {
   if (1<2) {
     return 5;
   }
   else {
     return true;
   }
 }

  /** This method is automatically generated by the Language Level Converter. */
  public TestingCrazyStuff() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      TestingCrazyStuff cast = ((TestingCrazyStuff) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
